public/locales/en-US.json

```diff
@@ -131,6 +132,7 @@
     "legendTooltip_Sockets_physical_RHEL for IBM Power": "$t(curiosity-graph.legendTooltip_Sockets_physical_RHEL)",
     "legendTooltip_Sockets_physical_RHEL for IBM z": "$t(curiosity-graph.legendTooltip_Sockets_physical_RHEL)",
     "legendTooltip_Sockets_physical_RHEL for x86": "$t(curiosity-graph.legendTooltip_Sockets_physical_RHEL)",
+    "legendTooltip_Sockets_physical_RHEL for x86-eus": "$t(curiosity-graph.legendTooltip_Sockets_physical_RHEL)",
```

Comment:
required: the string keys need to align with the product identifier, looks like it got overlooked or blended with the original `x86` and the one below `sap-x86`

should be
```
..._rhel-for-x86-eus
```

Reply:
updated